cms-adapter/src/Rsbc.Dmf.CaseManagement.Tests/Integration/CaseManagerTests.cs

```diff
@@ -140,7 +142,30 @@ public async Task CanUpdateNonComplyDocuments()
         }
 
 
-        
+        [Fact(Skip = RequiresDynamics)]
+        public async Task CanCreateBringForward()
+        {
+            var driverLicenseNumber = "111";
```

Comment:
111 is not a valid value for a DL - please adjust to use the value of a secret rather than a hard coded value

Reply:
Have Fixed the issue to get the DL from secrets.